build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
``contentFiles`` with ``copyToOuput=true`` sounds like what you want here. The file type can be anything. ``runtimes/{rid}/native`` on the other hand is only for native dlls.